src/path/sentry_path_windows.c

```diff
@@ -143,11 +198,27 @@ sentry__path_from_wstr_n(const wchar_t *s, size_t s_len)
     if (!s) {
         return NULL;
     }
-    sentry_path_t *rv = path_with_len(s_len + 1);
-    if (rv) {
-        memcpy(rv->path, s, s_len * sizeof(wchar_t));
-        rv->path[s_len] = 0;
+
+    sentry_path_t *rv = SENTRY_MAKE(sentry_path_t);
+    if (!rv) {
+        return rv;
     }
```

Comment:
We could be more consistent with the other 'failing' return paths here

```suggestion
    sentry_path_t *rv = SENTRY_MAKE(sentry_path_t);
    if (!rv) {
        return NULL;
}
```